README.md

```diff
@@ -1390,6 +1390,13 @@ I`m *[Hanna](https://github.com/HannaSyn)*, and I am really excited to join comm
 ![kitten](https://slm-assets.secondlife.com/assets/16934237/view_large/animals_05.jpg) 
 ---
 
-## What's up Kottans!
-My name is [Alex](https:/github.com/usides) and I want to be a dev-cat.
-![Image of DevCat](https://www.nexmo.com/wp-content/uploads/2016/08/dx-jamie-comic.jpg)
\ No newline at end of file
+
+## What's up Kottans! 
```

Comment:
Almost, but still affecting old changes. Rows from 1393 to 1395 shouldn't change